yarn.lock

```diff
@@ -145,11 +145,33 @@ argparse@^1.0.7:
   dependencies:
     sprintf-js "~1.0.2"
 
+array-differ@^2.0.3:
```

Comment:
It doesn't look like you are using any new dependencies. Why the changes in yarn.lock without any in package.json?

Reply:
Reverted.